### ISSUE TITLE
Week 2 announcements

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#9]
## The TLDR this week:

- [Week 2 todo](https://github.com/campus-experts/spring-2017/blob/master/todos/week-2.md)
- [Last weeks video](https://www.dropbox.com/s/3anmr8othdfa2vy/29thMarch2017Spring.mp4?dl=0), definitely watch the intros in the second half!
- Submit a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) with your community assessment. 
- Check out and comment on other people's [community assessments](https://github.com/campus-experts/spring-2017/pulls?q=is%3Apr+is%3Aopen+label%3Acommunity-assessment).
- Begin creating your [Community Impact Proposal](https://github.com/campus-experts/spring-2017/blob/master/docs/community-impact-proposal.md)

## Welcome to Week 2.

It's so exciting to have you folks join us! Your energy in the first calls and effort you put into your bios and profile pictures is amazing. I'm really excited to start reading your community assessments!

A couple of things I want to reiterate here:

- Your community assessments do not need to be perfect before you complete the pull request. These are living documents, and they'll change in response to feedback and developing ideas! Faster you get the PR in, faster everyone can chip in :sparkles: 
- Campus Experts is not a solo-mission. Ask for help, ask for feedback, from us at GitHub and your fellow experts. If you see another Expert is working on similar problems to you - collaborate! 
- Timelines and deadlines. You should aim to have your activity for the week done before you intend to start the next activity. This is because activities will often build on one another, and also to help prevent you getting snowed under!
- **It's okay to be busy**! You'll have exams, deadlines, hackathons, real life. If something comes up and you can't complete an exercise or attend a call, don't sweat it, but do let us know if you can!

Keep on adding your bios, speaker suggestions and community assessments! Reviews for CAs will happen asap. 

## This week on Slack
Missed a discussion on slack? Not to worry!

- In the UK or coming to the UK in May? We have tickets for London Satellite! Message Joe.
- In the US? [Come to Hackcon V](https://githubeducation.slack.com/archives/C4PF3VCSU/p1490958244104048)! 
- [Slack vs Facebook](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491244030301118) for student communities
- [Tools to analyse and support your community social networks](https://githubeducation.slack.com/archives/C2TQDRQSG/p1491311180372417)
